### PR TITLE
extra instructions for Hass.io add-on users

### PR DESCRIPTION
**Pull request by kdw2060**
Hi,

i had some trouble getting the integration to work on my Hass.io install with lots of trial and error. These instructions should help other users getting started more quickly.

cheers,
Kris

### CODE REVIEW

**Comment by kdw2060** [comment]
Hi, I was missing support for tasks, so tried to add support.
Be aware: I'm new to Python and also couldn't test as I haven't figured out where the pygrocy files that this component depends on are stored within the Hass.io filesystem. 

I basically copied the Chore-related classes etc and made Tasks-related ones.

Without having been able to test, there's probably still errors in this code. I would be happy to correct those, or off course feel free to do so yourself.

Also see my related pull request for pygrocy.

**Comment by SebRut** [comment]
Please submit a separate pull request for tasks integration.

**Comment by kdw2060** [comment]
Hi,

that's what I tried to do, but can't figure out the Github logic. When I choose to create a new pull request it adds the last commit to the list of commits for the previous pull request.
Can you accept the readme changes without the tasks integration? Or how can I remove the tasks commit from this pull request?

**Comment by SebRut** [comment]
You could undo changes introduced in a commit using `git revert`. The pull request contains both changes because you made all your changes on the same branch. You could try to 

1. checkout the latest version of this repo using `git checkout`
2. create a new branch for the changes you want to make `git checkout -b my_feature_name`
3. push that branch
4. create a PR for that branch

File: README.md [hunk 1]
```diff
@@ -22,5 +22,34 @@ grocy:
 7. Restart Home Assistant
 8. Look for the new Grocy sensor in States and use its info
 
+
+## Additional installation instructions for Hass.io users
+
+The configuration is slightly different for users that use Hass.io and the [official Grocy addon](https://github.com/hassio-addons/addon-grocy) from the official Hass.io Add-on store.
+
+1. If you haven't already done so, install Grocy from the Add-on store
+2. In the 'Network' section of the add-on config, input 9192 in the host field [screenshot](https://github.com/kdw2060/grocy/raw/develop/grocy-addon-config.png). Save your changes and restart the add-on.
```
Comment (SebRut) [inline README.md]: I guess this should point to this repo instead

Comment (kdw2060) [reply README.md]: done, i supposed the path should point to the master branch so used that

**Pull request merged**